src/specs/models/processing-config-bodies/ProcessingConfigMitochondrialContent.v1.yaml

```diff
@@ -24,9 +24,6 @@ properties:
                 type: number
 additionalProperties:
   type: object
-  required:
```

Comment:
I think I am missing some context here -- why are these no longer required?

Reply:
the actual required is inside the properties of `additionalProperties`, it was a bug that this required was here (requiring the base object to have these two properties).